doc/decisions/0_drafts/transformations.md

```diff
@@ -0,0 +1,152 @@
+# Transformations
+
+## Problem
+
+Certain plugins, e.g. `rename`, perform extensive transformations on keys and keysets within Elektra.
+Those transformations include, but are not limited to:
+
+- Changing the names of the keys inbetween `kdbGet` and `kdbSet` operations
+- Changing values back and forth for normalization, e.g. `true` -> `1`, `1` -> `true`
+- Adding and removing metadata
```

Comment:
Can you give an example here to make more clear why this is a transformation.

Reply:
Well I think we can leave out metadata completely from this decision.